README.md

```diff
@@ -1,136 +1,407 @@
-# Pre-requisites
-
-You need to make sure you have at least the followings to use the theme:
+# The LEAN Theme 
+## A no-frills WordPress starter theme
+
+![maintained](https://img.shields.io/badge/maintained-yes-brightgreen.svg) ![wordpress](https://img.shields.io/badge/wordpress-%3E%3D4.8-green.svg) ![php](https://img.shields.io/badge/php-%3E%3D7.0-green.svg) [![GitHub contributors](https://img.shields.io/github/contributors/wearenolte/lean-theme.svg)](https://github.com/wearenolte/lean-theme/graphs/contributors)
```

Comment:
What other software versions can we add here? I put WP 4.8 (is this right) and PHP 7.0. 
What about Node/NPM?
Anything else?

Reply:
I think It has to be PHP 7.1 at least

Reply:
Ok. Why 7.1 and not 7.0?